clar2wasm/src/words/consensus_buff.rs

```diff
@@ -37,13 +37,17 @@ impl ComplexWord for ToConsensusBuff {
             })?
             .clone();
 
-        // Save the offset (current stack pointer) into a local.
-        // This is where we will serialize the value to.
-        let offset = generator.module.locals.add(walrus::ValType::I32);
+        let expr_ty = generator
+            .get_expr_type(_expr)
```

Comment:
Since we now use `_expr`, we should remove the `_` in front of it.

Reply:
You are right, removed it.